.github/workflows/cla.yml

```diff
@@ -0,0 +1,52 @@
+name: "CLA Assistant"
+on:
+  issue_comment:
+    types: [created]
+  pull_request_target:
+    types: [opened, closed, synchronize]
+
+jobs:
+  CLAssistant:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "CLA Assistant"
+        if: |
+          (github.event.comment.body == 'recheck' ||
+           github.event.comment.body == 'I have read the CLA Document and
+           I hereby sign the CLA') ||
+           github.event_name == 'pull_request_target'
+        uses: cla-assistant/github-action@v2.1.3-beta
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PERSONAL_ACCESS_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
+        with:
+          path-to-signatures: 'signatures/version1/cla.json'
+          path-to-document: |
+           'https://github.com/cla-assistant/github-action/blob/master/SAPCLA.md'
```

Comment:
We shouldn't use this.
Take a look at Parity's https://cla.parity.io/sign_cla/20213174/3

Reply:
Sure. I will check with Parity's CLA.